addons/account/models/account_move_line.py

```diff
@@ -1715,16 +1718,15 @@ def _search_panel_domain_image(self, field_name, domain, set_count=False, limit=
         # Override in order to not read the complete move line table and use the index instead
         query = self._search(domain, limit=1)
         query.add_where('account.id = account_move_line.account_id')
-        query_str, query_param = query.select()
-        self.env.cr.execute(f"""
+        ids = self.env.get_select_result(SQL("""
             SELECT account.root_id
               FROM account_account account,
-                   LATERAL ({query_str}) line
+                   LATERAL (%s) line
              WHERE account.company_id IN %s
-        """, query_param + [tuple(self.env.companies.ids)])
+        """), query.select(), tuple(self.env.companies.ids))
```

Comment:
```suggestion
        """, query.select(), tuple(self.env.companies.ids)))
```
Arguments of the SQL(), not get_select_result()